addons/website_sale/models/website.py

```diff
@@ -666,3 +675,18 @@ def _get_checkout_steps(self, current_step=None):
         if current_step:
             return next(step for step in steps if current_step in step[0])[1]
         return steps
+
+class Menu(models.Model):
+    _inherit = "website.menu"
+
+    def _compute_visible(self): #mimic this
```

Comment:
Is this comment needed? Maybe it is better to put it in the docstring as well?
if needed then
```suggestion
    def _compute_visible(self):  # mimic this
```

Reply:
No, just forgot to delete this.